test_vm/vendor/cookbooks/dbfit_test/metadata.rb

```diff
@@ -7,7 +7,7 @@
 version          '0.3.0'
 depends          'mysql2_chef_gem'
 depends          'yum-mysql-community'
-depends          'mysql', '~> 8.2'
+depends          'mysql'
```

Comment:
Do we really need that change already? With `~> 8.2` we should be getting the latest 8.x >= 8.2.